app/controllers/admin/project_approvals_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Admin
+  # Allows a project to be approved
+  class ProjectApprovalsController < AdminController
```

Comment:
There's probably a way to do this as a ModeratorAction controller, where you include the `action` as a hidden field in the form that gets passed in as a query param or something. Maybe I'll do that when I add the rejection feature and putting a public project on hold.